guided-meditations.md

```diff
@@ -261,6 +261,13 @@ Algebra](algebraicguide.md).
 Computes the value _refexpr_ and if it is equal to any _textexpr_, the matching
 _step_ will be performed. If no value match, _altstep_ is performed instead.
 
+- `Fork` _name_ `In` _source_`:` [ _transforms_ ]\* `Title` _title_ _step_
+
+Splits the journey into several parallel journeys.  _source_ and _transforms_
```

Comment:
parallel journeys? That sounds to me like the other branch is executing in the background, is that correct?

Reply:
I'm not sure the best way to describe this. Let's discuss in the Shesmu meeting.